client/layout/guided-tours/config-elements/link-quit.js

```diff
@@ -33,10 +33,10 @@ export default class LinkQuit extends Component {
 
 	render() {
 		const { children, primary, subtle, href, target } = this.props;
-		const classes = subtle ? 'guided-tours__subtle-button' : '';
+		const classes = subtle ? ' guided-tours__subtle-button' : '';
 		return (
 			<Button
-				className={ classes }
+				className={ 'button-link' + classes }
```

Comment:
If you'd like to keep this component, let's namespace this using the [component name](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md):

```javascript
const classes = classNames( 'link-quit', classes, {
    subtle: 'is-subtle'
} );
//...
className={ classes }
```

Otherwise, let's update this in the button component itself. I'll ping design for 👀